platform-dom/src/main/java/com/softicar/platform/dom/elements/checkbox/DomCheckbox.java

```diff
@@ -207,4 +207,17 @@ public void setLabel(IDisplayString label) {
 			}
 		}
 	}
+
+	@Override
+	public IDomValueInput<Boolean> setRequired(boolean required) {
```

Comment:
DomCheckbox is not a native input nor checkbox, so the `required` attribute is completely useless.